config/dumps/config.env

```diff
@@ -1,5 +1,5 @@
 SPARQL_ENDPOINT=http://vfb-triplestore:8080/rdf4j-server/repositories/cap
 VFB_CONFIG=https://raw.githubusercontent.com/EBISPOT/cap-pipeline-config/main/config/prod/neo4j2owl-config.yaml
-DUMPS_SOLR="all preferred_roots deprecation_label image_names has_image"
+DUMPS_SOLR="Species Common_species"
```

Comment:
Should 'all' remain in this list?

Reply:
Yep.  Sorry for lack of doc.  I've updated the README to make this clearer https://github.com/EBISPOT/cap-pipeline-config/blob/main/config/dumps/README.md

Reply:
Updated file names to match case of labels